parm/config/gefs/config.wave

```diff
@@ -19,25 +19,11 @@ export RUNRSTwave="gdas"
 export waveGRDN='1'                   # grid number for ww3_multi
```

Comment:
waveGRDN and waveGRDG should be removed for gefs and gfs

Reply:
Also, there are a few places in this file where it says multi/shel, which I suppose we should go ahead and fix the comments for and remove multi